test/hclwrite/hclwrite.go

```diff
@@ -175,7 +178,13 @@ func AttributeValue(t *testing.T, name string, expr string) BlockBuilder {
 	}
 	body := res.Body.(*hclsyntax.Body)
 
-	val, diags := body.Attributes[name].Expr.Value(nil)
+	scope := &tflang.Scope{}
+	ctx := &hhcl.EvalContext{
+		Functions: scope.Functions(),
```

Comment:
so we can force an attribute to be a list using "tolist"..or else there is no way to create actual lists only tuples =P